eureka-common/src/main/java/edu/emory/cci/aiw/cvrg/eureka/common/props/AppPropertiesRegistration.java

```diff
@@ -0,0 +1,168 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package edu.emory.cci.aiw.cvrg.eureka.common.props;
+
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ *
+ * @author Miao Ai
+ */
+public class AppPropertiesRegistration {
+	private boolean localAccountRegistrationEnabled = false;
+	private boolean registrationEnabled = false;
+	private String googleOAuthKey;
+	private String googleOAuthSecret;
+	private String gitHubOAuthKey;
+	private String gitHubOAuthSecret;
+	private String twitterOAuthKey;
+	private String twitterOAuthSecret;
+	private String globusOAuthKey;
+	private String globusOAuthSecret;        
+	/**
+	 * Constructor.
+	 *
+	 */    
+	AppPropertiesRegistration(){
```

Comment:
Have a zero-arg constructor that does nothing, and a 1-arg constructor with a ServiceProperties parameter that sets the app properties from a ServiceProperties instance.